python/src/aiconfig/editor/client/src/shared/prompt_schemas/ClaudeBedrockPromptSchema.ts

```diff
@@ -0,0 +1,71 @@
+import { PromptSchema } from "../../utils/promptUtils";
+
+export const ClaudeBedrockPromptSchema: PromptSchema = {
+  // See https://docs.anthropic.com/claude/reference/complete_post
+  // for settings and defaults. The settings below are supported settings specified in the ClaudeBedrockModelParser
+  // refine_chat_completion_params implementation.
+  input: {
+    type: "string",
+  },
+  model_settings: {
+    type: "object",
+    properties: {
+      model: {
+        type: "string",
+      },
+      max_tokens_to_sample: {
+        type: "number",
+        description: `The maximum number of tokens to generate before stopping.
+        Positive values penalize new tokens based on their existing frequency in the text so far, decreasing the model's likelihood to repeat the same line verbatim.`,
+      },
+      stop_sequences: {
+        type: "array",
+        items: {
+          type: "string",
+        },
+        description: `Sequences that will cause the model to stop generating.`,
```

Comment:
Don't really understand this but that's fine, just from official documentation. As long as we have a good cookbook with local editor should be cool

Reply:
here's an example to help you visualize it
<img width="914" alt="Screenshot 2024-01-29 at 5 38 40 PM" src="https://github.com/lastmile-ai/aiconfig/assets/141073967/e2905004-3d1c-4625-b29a-9f77b2b6ef2f">